src/Tests/Microsoft.Diagnostics.Monitoring.Tool.UnitTests/AuthenticationTests.cs

```diff
@@ -287,8 +287,8 @@ private static byte[] GenerateApiKey()
         {
             byte[] apiKey = new byte[32]; // 256 bits
 
-            using RNGCryptoServiceProvider rngProvider = new();
-            rngProvider.GetBytes(apiKey);
+            using RandomNumberGenerator rng = RandomNumberGenerator.Create();
+            rng.GetBytes(apiKey);
```

Comment:
I ran into this issue as well. VS recommended: RandomNumberGenerator.Fill(new Span<byte>(apiKey));